pkg/controllers/node/termination/terminator/terminator.go

```diff
@@ -113,9 +101,6 @@ func (t *Terminator) Evict(pods []*v1.Pod) {
 	// 1. Prioritize noncritical pods, non-daemon pods https://kubernetes.io/docs/concepts/architecture/nodes/#graceful-node-shutdown
 	var criticalNonDaemon, criticalDaemon, nonCriticalNonDaemon, nonCriticalDaemon []*v1.Pod
 	for _, pod := range pods {
-		if !pod.DeletionTimestamp.IsZero() {
-			continue
-		}
```

Comment:
Why are we skipping this now? Seems like we'll be enqueuing more eviction calls now, which I think is redundant.

Reply:
It's ruled out during the "evictablePods" check